### DISCUSSION TITLE
Documentation - Focus more on actual technology names rather than product names

### DISCUSSION THREAD

**Discussion by fineon** [laravel/framework#44561] [General]
Hi all,
I'm new to Laravel and reading through the documentation to assess a fit for my team. I noticed it took me a while to get a sense of what Laravel can do since the documentation emphasizes Laravel brand names rather than the actual technology behind it. Like how Blade is Laravel's templating language. Most other frameworks still focus on the task and technology topics. For example: a quick start guide can mention about templating first, then provide the solution like using Blade. [Vue.js documentation](https://vuejs.org/guide/essentials/template-syntax.html) is a good example of this.

Just putting my thoughts out loud and see if this is a direction the docs team could find useful. Thanks

**Comment by Treggats** [comment]
Personally I think the docs are pretty good and not brand oriented, just my two cents.
In the sense of a more practical approach, have you looked at https://bootcamp.laravel.com/?
